unstract/connectors/src/unstract/connectors/databases/bigquery/bigquery.py

```diff
@@ -88,22 +90,71 @@ def sql_to_db_mapping(value: str) -> str:
         }
         return mapping.get(python_type, "string")
 
-    @staticmethod
-    def get_create_table_query(table: str) -> str:
+    def get_create_table_base_query(self, table: str) -> str:
+        """Function to create a base create table sql query.
+
+        Args:
+            table (str): db-connector table name
+
+        Returns:
+            str: generates a create sql base query with the constant columns
+        """
+        bigquery_table_name = str.lower(table).split(".")
+        if len(bigquery_table_name) != self.big_query_table_size:
+            raise ValueError(
+                f"Invalid table name format: '{table}'. "
+                "Please enter correct correct bigquery table in the form "
+                "{table}.{schema}.{database}."
+            )
         sql_query = (
             f"CREATE TABLE IF NOT EXISTS {table} "
             f"(id string,"
             f"created_by string, created_at TIMESTAMP, "
         )
         return sql_query
 
+    @staticmethod
+    def get_sql_insert_query(table_name: str, sql_keys: list[str]) -> str:
+        """Function to generate parameterised insert sql query.
+
+        Args:
+            table_name (str): db-connector table name
+            sql_keys (list[str]): column names
+
+        Returns:
+            str: returns a string with parameterised insert sql query
+        """
+        keys_str = ",".join(sql_keys)
+        values_placeholder = ",".join(["@" + key for key in sql_keys])
+        return f"INSERT INTO {table_name} ({keys_str}) VALUES ({values_placeholder})"
+
     def execute_query(
         self, engine: Any, sql_query: str, sql_values: Any, **kwargs: Any
     ) -> None:
+        """Executes create/insert query.
+
+        Args:
+            engine (Any): big query client engine
+            sql_query (str): _description_
+            sql_values (Any): _description_
+
+        Raises:
+            BigQueryForbiddenException: _description_
+            BigQueryNotFoundException: _description_
+            ColumnMissingException: _description_
+        """
         table_name = str(kwargs.get("table_name"))
```

Comment:
Can we add a `None` check here? If `table_name` is None in `kwargs`, it will be converted to the string `"None"`, which may not be the intended behavior.